libs/partners/pinecone/tests/integration_tests/test_vectorstores.py

```diff
@@ -8,7 +8,7 @@
 import pytest  # type: ignore[import-not-found]
 from langchain_core.documents import Document
 from langchain_openai import OpenAIEmbeddings  # type: ignore[import-not-found]
-from pinecone import PodSpec
+from pinecone import PodSpec  # type: ignore
```

Comment:
why does this need a type ignore?

Reply:
Removed and dropped on other ignore that was resolved